guava/src/com/google/common/collect/Iterators.java

```diff
@@ -1135,6 +1136,139 @@ public T nextElement() {
   }
 
   /**
+   * Create an possibly infinite Iterator that uses the provided Generator
```

Comment:
According to the javadoc standard in https://google-styleguide.googlecode.com/svn/trunk/javaguide.html, it should be 'Creates' and not Create'. This will also be consistent with other methods of this class.